ui/raidboss/data/02-arr/dungeon/aurum_vale.ts

```diff
@@ -0,0 +1,33 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheAurumVale,
+  triggers: [
+    {
+      id: 'Aurum Vale GoldLungs/Burrs',
+      type: 'GainsEffect',
+      // Count `0[2-9]` here is to filter out the first stack
+      netRegex: NetRegexes.gainsEffect({ effectId: ['12E', '12F'], count: '0[2-9]' }),
```

Comment:
Isn't it eat after 2 on the first boss, eat after 3 on the final boss, or am I misremembering?

Reply:
TBH, I'm not 100% sure, but I don't see a real downside to eating it at 2+ on both, nor ever ran into not having enough fruits. Can be changed if you want though